chromium expects clang libs in:
  $clangprefix/lib/clang/$clangver/lib/$_dir/libclang_rt.whatever$_suffix.a
and for some reason the computed paths, which are like:
  /usr/lib/clang/16/lib/$arch-unknown-linux-gnu/libclang_rt.whatever.a
are totally wrong - on alpine the libs are actually at:
  /usr/lib/clang/16/lib/linux/libclang_rt.whatever-$arch.a
--- a/build/config/clang/BUILD.gn
+++ b/build/config/clang/BUILD.gn
@@ -116,14 +116,15 @@
       } else if (is_apple) {
         _dir = "darwin"
       } else if (is_linux || is_chromeos) {
+        _dir = "linux"
         if (current_cpu == "x64") {
-          _dir = "x86_64-unknown-linux-gnu"
+          _suffix = "-x86_64"
         } else if (current_cpu == "x86") {
-          _dir = "i386-unknown-linux-gnu"
+          _suffix = "-i386"
         } else if (current_cpu == "arm") {
-          _dir = "armv7-unknown-linux-gnueabihf"
+          _suffix = "-armv7"
         } else if (current_cpu == "arm64") {
-          _dir = "aarch64-unknown-linux-gnu"
+          _suffix = "-aarch64"
         } else {
           assert(false)  # Unhandled cpu type
         }
